src/assets/css/dashboard.css

```diff
@@ -291,6 +287,7 @@ div[data-role=controlgroup] a.ui-btn-active {
 }
 
 .activeSession {
+    min-width: 15rem;
```

Comment:
```suggestion
    min-width: 20rem;
```

Reply:
15rem is still a bit narrow on iPad safari.

15rem<img src="https://user-images.githubusercontent.com/14953024/102685124-2c502e80-4219-11eb-8bbb-f7f08c79ff33.jpg" width = "200" align=center />
20rem<img src="https://user-images.githubusercontent.com/14953024/102685129-2fe3b580-4219-11eb-9afe-19a50f4e8b2b.jpg" width = "200" align=center />